content/200-concepts/100-components/03-prisma-migrate/200-shadow-database.mdx

```diff
@@ -125,7 +125,7 @@ Database error: Error querying the database: db error: ERROR: permission denied
 To resolve this error:
 
 - If you are working locally, we recommend that you update the database user's privileges.
-- If you are developing against a cloud-based database (for example, on Heroku and Digital Ocean) see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually).
-- If you are developing against a cloud-based database (for example, on Heroku and Digital Ocean) and are currently **prototyping** such that you don't care about generated migration files and only need to apply your Prisma data model to the database schema, you can run [`prisma db push`](/reference/api-reference/command-reference#db) instead of the `prisma migrate dev` command.
+- If you are developing against a cloud-based database (for example, on Heroku, Digital Ocean and Vercel Postgres) see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually).
+- If you are developing against a cloud-based database (for example, on Heroku, Digital Ocean and Vercel Postgres) and are currently **prototyping** such that you don't care about generated migration files and only need to apply your Prisma data model to the database schema, you can run [`prisma db push`](/reference/api-reference/command-reference#db) instead of the `prisma migrate dev` command.
```

Comment:
```suggestion
- If you are developing against a cloud-based database (for example, on Heroku, Digital Ocean, or Vercel Postgres) see: [Cloud-hosted shadow databases](#cloud-hosted-shadow-databases-must-be-created-manually).
- If you are developing against a cloud-based database (for example, on Heroku, Digital Ocean, or Vercel Postgres) and are currently **prototyping** such that you don't care about generated migration files and only need to apply your Prisma data model to the database schema, you can run [`prisma db push`](/reference/api-reference/command-reference#db) instead of the `prisma migrate dev` command.
```